h2o-genmodel/src/main/java/hex/genmodel/attributes/ModelJsonReader.java

```diff
@@ -392,7 +401,8 @@ private static Object convertJsonObject(final JsonObject convertFrom) {
                 convertFrom.get("value").getAsDouble()
             );
         } else {
-            throw new UnsupportedOperationException(String.format("Object not supported: \n %s ", convertFrom.toString()));
+            LOG.warn(String.format("Error reading MOJO JSON: Object not supported: \n %s ", convertFrom.toString()));
```

Comment:
nit: Shouldn't this case be logged as error?

Reply:
fixed